Update CHANGELOG for v2025.07.08 release

satyanandak opened the pull request:
Please let me know of any important changes that you want to make visible by adding to the Change log.

ujohnny commented:
@satyanandak please note that I've also updated changelog in v2025.06.24 as I've promoted some fixes to it

satyanandak commented:
> @satyanandak please note that I've also updated changelog in v2025.06.24 as I've promoted some fixes to it

Noted !
updated the CHANGELOG, can you please review and approve ?
